Cron/OrderSyncStatus.php

```diff
@@ -41,6 +41,7 @@ class OrderSyncStatus
         "omise_offsite_maybankqr",
         "omise_offsite_shopeepay",
         "omise_offsite_atome",
+        "omise_offsite_paypay",
```

Comment:
Why can't new payment methods register themselves?

Reply:
We can. Original design was not designed to be able to register automatically.
For now, we have to register manually unless we change the structure.

Reply:
Great, thanks.